### DISCUSSION TITLE
Collection list UI/UX (collapsible)

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#17682] [Feature Requests]
### Summary

By default all folders and collections are opened in the sidebar.
For folders we can set that they should start either:
- Start open
- Start collapsed
- Always stay open

This setting can be changed by clicking on the folder-Icon what I think is not really intuitive (just discovered it right now after two years with directus - not sure if this always existed 😅)
![Bildschirm­foto 2023-03-02 um 14 55 58](https://user-images.githubusercontent.com/46492597/222449164-f7569221-e7e4-4335-8428-8a642733abc5.png)

I'd suggest the following changes:
- Using a dropdown to select the behaviour
- More distinct icons (as you can see in the screenshot above the icons are nearly identical)
- Add the setting to the edit-folder modal (as it's called "Edit folder" I'd expect that it shows me all options)

And as a new feature:
Right now this behaviour can't be set for bookmarks. Would be great if they could be collapsed by default, too

### Motivation

Increase the UI / UX for the App, make the settings more intuitive.

### Detailed Design

see above

### Must Have:
- More distinctive icons
- Dropdown to select the behavious

### Should Have:
- Move it into the settings modal

### Could Have:
- Setting for Bookmarks to start collapsed too

### Drawbacks

/

### Alternatives

/

### Adoption Strategy

/

**Comment by pavelstianko** [comment]
I agree that changing the setting by clicking on the icon isn't  very intuitive. Also It would be nice to have possibility to collapse folder at Data model page as well - If you have large model with a lot of folders it is hard to find the right collection.

**Comment by that1matt** [comment]
I created a feature branch, and hopefully this gets implemented into Directus. [Post](https://github.com/directus/directus/discussions/9939#discussioncomment-5833036)

**Reply by rijkvanzanten** [reply]
I believe this discussion is asking specifically to turn that three-way click into a dropdown, rather than adding a collapse toggle for use on that page 🙂 

I've opened an issue for that piece particularly on https://github.com/directus/directus/issues/18517

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/18518
